pkgs/development/ruby-modules/with-packages/Gemfile

```
@@ -0,0 +1,71 @@
+source 'https://rubygems.org' do
+  gem 'addressable'
+  gem 'awesome_print'
+  gem 'bacon'
+  gem 'byebug'
+  gem 'bundler'
+  gem 'camping'
+  gem 'capybara-webkit'
+  gem 'coderay'
+  gem 'concurrent-ruby'
+  gem 'daemons'
+  gem 'digest-sha3'
+  gem 'domain_name'
+  gem 'equipment'
+  gem 'eventmachine'
+  gem 'excon'
+  gem 'faraday'
+  gem 'ffi'
+  gem 'hashie'
+  gem 'highline'
+  gem 'hike'
+  gem 'hpricot'
+  gem 'httpclient'
+  gem 'http-cookie'
+  gem 'jbuilder'
+  gem 'jmespath'
+  gem 'jwt'
+  gem 'magic'
+  gem 'markaby'
+  gem 'metaid'
+  gem 'method_source'
+  gem 'mini_magick'
+  gem 'msgpack'
+  gem 'mysql'
+  gem 'mysql2'
+  gem 'ncursesw'
+  gem 'netrc'
+  gem 'net-scp'
+  gem 'net-ssh'
```

```suggestion
  gem 'net-ssh'
  gem 'nokogiri'
```
you forgot the most important one :p